src/data/members.js

```diff
@@ -18,6 +18,7 @@ module.exports = {
 	jn222na: require("./users/jn222na.json"),
 	Pajn: require("./users/Pajn.json"),
 	ek222mw: require("./users/ek222mw.json"),
-	me222rs: require("./users/me222rs.json")
+	me222rs: require("./users/me222rs.json"),
+	angamanga: require("./users/angamanga.json")
```

Comment:
Here we must make `annamanga` upper-case, so the line reads like: `Angamanga: require("./users/Angamanga.json")`